server/lib/routes.js

```diff
@@ -129,7 +129,10 @@ module.exports = function (config, i18n) {
       '/oauth/': true,
       '/oauth/force_auth': true,
       '/oauth/signin': true,
-      '/oauth/signup': true
+      '/oauth/signup': true,
+      '/reset_password': true,
```

Comment:
I added reset_password as well because it's easy to get to from the signin page.